cmd/root.go

```diff
@@ -18,6 +18,10 @@ var rootCmd = &cobra.Command{
 	Short: "Microsoft Azure CLI kubectl plugin",
 }
 
+func init() {
+	utils.InitFlags()
+}
```

Comment:
I don't have an answer but what is the advantage of calling it from `init` instead of doing it from `Execute`? Is it more go-style?